apps/web/src/components/honors-list/index.tsx

```diff
@@ -0,0 +1,77 @@
+import type { AwardedHonor, Honor } from "@tietokilta/cms-types/payload";
+import { ChevronDownIcon } from "@tietokilta/ui";
+import _ from "lodash";
+
+function AwardedPersonDropdown({
```

Comment:
This could be made into a generic component, like `Collapse`, in case we would want to use such a component elsewhere. But I suppose this is fine until someone actually needs it :D

Reply:
yes let's do this later if needed ":D"